src/renderer/components/atoms/button/button.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react/button-has-type */
+import type { ButtonProps } from './types';
+import { buildStylesFor } from './helper';
+
+const Button = ({
+  className,
+  label,
+  type = 'primary',
```

Comment:
what if we name this prop `as` instead of `type`? I think it would be more consistent with the `Text` component. Also the `type` prop in buttons is used to specify what role the button is accomplishing in a form
https://www.w3schools.com/tags/att_button_type.asp

Reply:
agree, done!